### PR TITLE
Install compatible PHPUnit version and use it on Travis CI

### PR DESCRIPTION
**Pull request by xabbuh**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -9,3 +9,6 @@ php:
 
 before_script:
   - composer install
+  
+script:
+  - ./vendor/bin/phpunit
```
Comment (BenjaminPaap) [inline .travis.yml]: This should be `./bin/phpunit` because of composer's `bin-dir` setting, shouldn't it?

Comment (monofone) [inline .travis.yml]: Yes, you are right thtas also why the ci fails to build.

**Comment by BenjaminPaap** [comment]
Thanks for your contribution :+1:

**Comment by xabbuh** [comment]
Thanks

**Pull request merged**